cmd/podman/shared/pod.go

```diff
@@ -134,4 +137,129 @@ func CreatePortBindings(ports []string) ([]ocicni.PortMapping, error) {
 	return portBindings, nil
 }
 
+func GetPodsWithFilters(r *libpod.Runtime, c *cliconfig.PodPsValues) ([]*libpod.Pod, error) {
+
+	var pods []*libpod.Pod
+	var err error
+	if c.Latest {
```

Comment:
I don't know if this should be part of it - @baude Thoughts?

Reply:
Should I put it back in pod_ps? So, Currently, it's not like that, but do we have to apply pods filter on the latest fetched pod as well if so we will need the flag here.

Reply:
Filters probably should not affect --latest - in fact, I'd go so far as to say filters and `--latest` should conflict.